README.md

```diff
@@ -46,3 +46,9 @@ cluster, and debug your running app in seconds!
 [10]: https://cloud.google.com/code/docs/vscode/install
 [11]: https://cloud.google.com/code/docs/vscode/troubleshooting
 [12]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/app_deploy_debug.gif
+
+## Support
+
+- Join our community of users using Slack channel [cloud-code-ide](https://googlecloud-community.slack.com/messages/cloud-code-ide) for Cloud Code support and outreach.
+- To report any bug or task on the product, continue to use github issues, which will be triaged internally and updated.
+- If you are not sure of it is either a bug or product ask, to reach out to cloud code users community, create a Stack Overflow question, and add the two tags [google-cloud-platform] and [cloud-code] to your question, and post a link to it in our Slack channel.
```

Comment:
When users are looking for solutions, is it easier for them to look for Cloud Code solutions or is it easier to look for Cloud Code for IntelliJ vs. Cloud Code for VS Code solutions? What is the thought process behind a single tag vs. two different tags vs. multiple tags here on StackOverflow?